crates/frontend/src/pages/dimensions.rs

```diff
@@ -173,7 +204,13 @@ pub fn dimensions() -> impl IntoView {
                         </div>
                     }
                 }}
-
+            <DeleteModal
+                modal_visible=delete_modal_visible
+                confirm_delete=confirm_delete
+                set_modal_visible=set_delete_modal_visible
```

Comment:
ideally this should have been an on_close prop, because a modal can never set itself as visible, it can only close itself, and the prop should be called in all those places where the Modal wants to trigger a close action, and then the on_close prop passed from here should do whatever is needed with respect to on_close action.

The caller should not have to rely on what is being sent from the modal to this function